src/Message/HashFile.php

```diff
@@ -10,7 +10,7 @@ class HashFile
     /**
      * The file ID of the to be hashed file.
      *
-     * @var string
+     * @var integer
```

Comment:
I've always left it a string unless doing math with it. That was a very old rule though, so maybe time to discuss again.

Reply:
I think its better we use integers for IDs, unless it is like UDI or DOI